services/librariesio/librariesio-base.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+const { nonNegativeInteger } = require('../validators.js')
+
+// API doc: https://libraries.io/api#project
+const schema = Joi.object({
+  dependents_count: nonNegativeInteger,
+  dependent_repos_count: nonNegativeInteger,
+}).required()
+
+class LibrariesIoBase extends BaseJsonService {
```

Comment:
Hmm.. one more thing: Thinking a step ahead, when we port over the [other libraries io badges](https://github.com/badges/shields/blob/db4bffb300a8d9c794fbd089a2f0f9586415e500/server.js#L6516-L6578) we'll also need to be able to pass another param (version/latest) which is not relevant here and we'll also call a different API endpoint, so probably a single `LibrariesIoBase` class which hard-codes this stuff isn't going to be the ideal abstraction.

Maybe we just cross that bridge when we come to it, but maybe worth considering now?

Reply:
Yea! Initially I started writing that in here, but stopped because I didn't want to add code that had no use or a test. We could add a second parameter `{ withVersion }` like NpmBase has.

Reply:
> I didn't want to add code that had no use or a test

:+1:  Lets deal with that case when we actually refactor the service that needs this